tests/test_image_onnx_embeddings.py

```
@@ -38,80 +65,69 @@ def test_embedding(model_name: str) -> None:
 
         dim = model_desc.dim
 
-        model = ImageEmbedding(model_name=model_desc.model)
+        with model_cache(model_desc.model) as model:
+            images = [
+                TEST_MISC_DIR / "image.jpeg",
+                str(TEST_MISC_DIR / "small_image.jpeg"),
+                Image.open((TEST_MISC_DIR / "small_image.jpeg")),
+                Image.open(BytesIO(requests.get("https://qdrant.tech/img/logo.png").content)),
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)